invites.go

```diff
@@ -0,0 +1,55 @@
+package rollbar
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type ListInvitesResponse struct {
+	Error  int `json:err`
+	Result []struct {
+		Id           int    `json:"id"`
+		FromUserId   int    `json:"from_user_id"`
+		TeamId       int    `json:"team_id"`
+		ToEmail      string `json:"to_email"`
+		Status       string `json:"status"`
+		DateCreated  int    `json:"date_created"`
+		DateRedeemed int    `json:"date_redeemed"`
+	}
+}
+
+func (s *Client) ListInvites(teamID int) (*ListInvitesResponse, error) {
+	var data ListInvitesResponse
+
+	// Invitation call has pagination.
+	// There's a feature request to expire the invitations after some time.
+	// Looping until we get an empty invitations list [].
+	// Page=0 and page=1 return the same result that's why we start from page=1
+	for i := 1; ; i++ {
+		pageNum := i
+		url := fmt.Sprintf("%steam/%d/invites?access_token=%s&page=%d", s.ApiBaseUrl, teamID, s.ApiKey, pageNum)
```

Comment:
It's weired that we have to hand in the API base url and access token in every method (ListInvites, ListUsers, InviteUser) this should be done in the client whose members these are.

Reply:
Ideally, the URL params would be build using the [URL package](https://golang.org/pkg/net/url). This could be done in the Client and we would only hand in the key value pairs. Perhaps, this is for a separate PR refactoring the Client.